src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -363,4 +363,7 @@
   <data name="BreakIntoDebuggerCommandKey" xml:space="preserve">
     <value>Break into the debugger</value>
   </data>
-</root>
+  <data name="OpenSettingsUICommandKey" xml:space="preserve">
+    <value>Open Settings UI</value>

```

Comment:
nit: maybe just.. `Open Settings`

UI is a term we use, not a term our community uses.